src/main/java/com/tlswe/awsmock/ec2/model/AbstractMockEc2Instance.java

```diff
@@ -96,21 +180,101 @@
          */
         C1_XLARGE("c1.xlarge"),
         /**
+         * c3.large.
+         */
+        C3_LARGE("c3.large"),
+        /**
+         * c3.xlarge.
+         */
+        C3_XLARGE("c3.xlarge"),
+        /**
+         * c3.2xlarge.
+         */
+        C3_2XLARGE("c3.2xlarge"),
+        /**
+         * c3.4xlarge.
+         */
+        C3_4XLARGE("c3.4xlarge"),
+        /**
+         * c3.8xlarge.
+         */
+        C3_8XLARGE("c3.8xlarge"),
+        /**
+         * c4.large.
+         */
+        C4_LARGE("c4.large"),
+        /**
+         * c4.xlarge.
+         */
+        C4_XLARGE("c4.xlarge"),
+        /**
+         * c4.2xlarge.
+         */
+        C4_2XLARGE("c4.2xlarge"),
+        /**
+         * c4.4xlarge.
+         */
+        C4_4XLARGE("c4.4xlarge"),
+        /**
+         * c4.8xlarge.
+         */
+        C4_8XLARGE("c4.8xlarge"),
+        /**
          * cc1.4xlarge.
          */
-        CC1_4XLARGE("cc1.4xlarge"),
+        CC_14XLARGE("cc1.4xlarge"),
```

Comment:
Typo? Why CC_14XLARGE rather than CC1_4XLARGE here? And fix the following a few similar occurrences too. 
But thanks for completing the latest inst types!